src/explore-education-statistics-frontend/src/modules/find-statistics/FindStatisticsPage.tsx

```diff
@@ -38,7 +38,10 @@ class FindStatisticsPage extends Component<Props> {
     const { themes } = this.props;
 
     return (
-      <Page breadcrumbs={[{ name: 'Find statistics and data' }]}>
+      <Page
+        breadcrumbs={[{ name: 'Find statistics and data' }]}
+        pageMeta={{ title: 'Find statistics and data' }}
```

Comment:
There's a lot of duplication of the page title across the breadcrumbs, `PageTitle` component and the breadcrumbs. I think ideally we need to abstract over all of these so that we can just provide `Page` with a `title` prop and it adds the title to all of the relevant automatically.

Reply:
This should be better, I've also stripped out the html page headers in various pages